.github/workflows/check-abi.yml

```diff
@@ -108,19 +107,18 @@ jobs:
       run: |
         mkdir base/build
         cd base/build
-        cmake -DCMAKE_BUILD_TYPE=Debug ..
+        cmake -E env CXXFLAGS="-Og -g" cmake -DCMAKE_BUILD_TYPE=Debug ..
         make -j$(nproc)
     - name: Build debug head
       run: |
         mkdir head/build
         cd head/build
-        cmake -DCMAKE_BUILD_TYPE=Debug ..
+        cmake -E env CXXFLAGS="-Og -g" cmake -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
Why is cmake being passed as an argument to cmake?

Reply:
This is because a change in the cmake to "add" new flags will not fix the older commit hash that needs to have these flags as well. The ABI Dump tool "requires" that you set these flags otherwise it cannot check the library files and read the symbols. This is a method to pass those flags to the cmake without having to modify the cmake to add the needed flags.

I can confirm that this correctly passes the flags to the cmake build such that both head and base library builds correctly have the needed flags for the ABI check.